packages/account/src/Components/poi/idv-document-submit/utils.js

```diff
@@ -128,4 +128,11 @@ const idv_document_data = {
             sample_image: '',
         },
     },
+    ug: {
+        national_id: {
+            new_display_name: 'National IDs',
+            example_format: 'CM37173109PE1D',
```

Comment:
maybe we should leave here number example accordingly to existing. for example CM12345678PE1D.